### ISSUE TITLE
Details about wav2vec2.0 pretraining hyper parameter ?

### ISSUE THREAD

**Issue by mujhenahiata** [facebookresearch/fairseq#4900]
What hyperparameters to look for when pretraining a wav2vec2 model.
when can one tell that the model is sufficiently trained enough, and by looking at what parameters, 
what  range will it be for those parameter, to say  that model is now trained enough.

**Comment by mujhenahiata** [comment]
@alexeib please help

**Comment by alexeib** [comment]
it is not really possible to tell if the model is "sufficiently trained" without evaluating it on some downstream task. you could implement periodic evaluation of checkpoints for instance. however that is still not great because we tend to use annealing learning rate schedules, and the models improve significantly when they are exposed to lower learning rates. the best advice i have is to train a few models for various lengths and see when it stops improving

as for hyper-parameters, we have a good starting set up of parameters in our config files (which the example in the README points to)
